app/controllers/landable/api/access_tokens_controller.rb

```diff
@@ -42,6 +48,18 @@ def find_own_access_token(id = params[:id])
       def asset_token_params
         params.require(:access_token).permit(:username, :password)
       end
+
+      def determine_permissions(user_groups) # input: ["HipChat Users", "PagerDuty Users", "popops"]
+        yaml_groups = YAML.load(File.read(Rails.root.join('config', 'ldap.yml')))[:permissions]["CNU"]
+
+        user_groups.inject([]) do |permissions, group|
+          permissions << "read" if yaml_groups[group]["read"]
+          permissions << "edit" if yaml_groups[group]["edit"]
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.